doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,131 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
 printer_health_offline		| **Optional.** The maximum number of seconds since the last update of cache file before it is considered too old.
 printer_health_multiline	| **Optional.** Multiline output
 
+#### Thola <a id="plugin-contrib-command-thola"></a>
+
+The [Thola](https://thola.io) plugin
+is a tool for monitoring network devices, that mainly uses SNMP.
+To monitor devices the subcommand `thola check` can be used.
+
+`Templates:`
+
+If you specify the Thola API address, snmp community and version in the following templates, then they will be 
+automatically used for each Thola subcommand.
+
+**generic-thola-check-command:**
+
+Name                    | Description
+------------------------|------------------------------------------------------
+thola_api_address       | Address of the Thola API to connect to
+
+**generic-thola-device-check-command:**
+
+Name                    | Description
+------------------------|------------------------------------------------------
+snmp_community          | SNMP community of the device
+snmp_protocol           | SNMP version to use
```

Comment:
And e.g. these?

Reply:
Oh yeah right, I didnt think about the documentation, I will change it as soon as possible